reactome_analysis_api/reactome_analysis_api/controllers/datasets_controller.py

```diff
@@ -43,6 +43,9 @@ def get_examples():  # noqa: E501
         ExternalData(id="EXAMPLE_SC_B_CELLS", title="B cell scRNAseq example", type="rnaseq_counts",
                      description="Single-cell RNA-seq data of B cells extracted from the Jerby-Arnon at al. study (Cell 2018).",
                      group="SC_EXAMPLES"),
+        ExternalData(id="EXAMPLE_METABOLOMICS", title="Liver cirrhosis metabolomics example", type="metabolomics",
+                     description="MTBLS5665: Transcriptomic and metabolomic analysis of liver cirrhosis",
```

Comment:
Maybe it would be better to not start the description with the identifier. It should follow the same structure as other example definitions in my opinion. (Not a bad idea to add the dientifier though, but maybe more at the end, it's more inviting to read)

Reply:
Done